resources/js/Shared/Components/Panel.vue

```diff
@@ -0,0 +1,69 @@
+<script setup>
+import Map from './Map.vue'
+import { ref, onMounted, onBeforeUnmount, watch } from 'vue'
+import Info from './Info.vue'
+import SearchPanel from './SearchPanel.vue'
+import Nav from './../Layout/Nav.vue'
+import Footer from '../Layout/Footer.vue'
+
+const showInfo = ref(true)
+const isMobile = ref(window.innerWidth <= 1024)
+const showMapMobile = ref(false)
+
+function switchPanel() {
+  showInfo.value = !showInfo.value
+}
+
+function switchMap() {
+  showMapMobile.value = !showMapMobile.value
+}
+
+onMounted(() => {
+  window.addEventListener('resize', updateIsMobile)
+})
+
+onBeforeUnmount(() => {
+  window.removeEventListener('resize', updateIsMobile)
+})
+
+function updateIsMobile() {
+  isMobile.value = window.innerWidth <= 1024
+
+  if (!isMobile.value) {
+    showMapMobile.value = false
+  }
+}
+
+watch(() => window.innerWidth, updateIsMobile)
+</script>
+
+<template>
+  <div class="mx-auto h-screen flex-col bg-white">
+    <Nav class="z-30 max-h-28" />
+    <div class="relative flex  min-h-[calc(100%-116px)] flex-col overflow-auto lg:flex-row">
+      <div class=" lg:w-1/2" :class="{'hidden': showMapMobile}">
+        <Info v-show="showInfo" @try-it-out="switchPanel" />
+        <SearchPanel v-show="!showInfo" />
+      </div>
+
+      <div v-if="!(isMobile && !showMapMobile)" class=" lg:w-1/2">
+        <Map class="z-10" />
+      </div>
+
+      <div class="flex justify-center">
+        <button v-if="!showInfo && isMobile" class="fixed bottom-5 z-20 flex items-center justify-center rounded-full bg-[#527ABA] px-2 py-1.5 text-sm font-semibold text-white shadow-sm hover:brightness-90 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600" @click="switchMap">
```

Comment:
Maybe add this color `bg-[#527ABA]` to `tailwind.config.js` file

Reply:
I forgot about changing these values after adding custom colors to tailwind confg. Already fixed, thanks.